app/assets/javascripts/components/company_overview.es6.jsx

```diff
@@ -0,0 +1,73 @@
+class CompanyOverview extends React.Component {
+  render () {
```

Comment:
you can add something like that
```
const { company, logo, pages_path, url_domain, industries } = this.props;
```
and then below remove almost all ```this.props.```

Reply:
actually, component data should be in `state`